GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -137,6 +146,22 @@ - (nullable NSString *)openIDRealm {
   return [_authState.lastTokenResponse.request.additionalParameters[kOpenIDRealmParameter] copy];
 }
 
+- (NSString *)lastTokenResponseToken {
```

Comment:
The naming for these four helper methods should be consistent and clearly communicate what they are doing.

Reply:
Done.